invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less

```diff
@@ -94,6 +101,19 @@ nav {
   display: block;
 }
 
+.active .navbar-item.navbar-button.navbar-actions-container {
```

Comment:
What happens if a new submenu is created dynamically via `flask-menu`?

Reply:
It should not interfere with the existing or new submenu's styling as everything scoped